src/watch/index.ts

```diff
@@ -21,46 +21,44 @@ import { addTask, deleteTask } from './fileWatchers';
 const DELAY = 100;
 
 export class Watcher extends EventEmitter {
-	dirty: boolean;
-	running: boolean;
-	tasks: Task[];
-	succeeded: boolean;
+	private buildTimeout: NodeJS.Timer;
+	private running: boolean = false;
```

Comment:
Any reason why you are initializing to false here but than switch to true in the constructor which would override this?